backend/data_preparation/crawler/twitter_crawler.py

```diff
@@ -81,6 +61,16 @@ def crawl(self, keywords: List, batch_number: int, fetch_from_db: bool) -> Union
 
             # gets status with the list that has batch number (can be a bit more than the batch#) amount of tweets
             ids = list(self.crawled_id_set)
+        elif using_sample_api:
+            ids = []
+            for t in (self.api.GetStreamSample()):
```

Comment:
actually, for sample API, I believe I can take in arguments like the keyword list. So that the API can do the filtering. Can you look into this please?

Reply:
The GetStreamSample does not support it. There is one API that can do the filtering: GetStreamFilter. The difference between these two is that GetStreamSample returns a small random sample of all public statuses and GetStreamFilter does not sample.

Reply:
So if it does not sample, isn't GetStreamFilter better? Our goal is to get as many records related to keywords as possible. Please elaborate.